Unable to reproduce the SQuAD v1.1 result with the BERT-Large PyTorch model.

I tried to reproduce the SQuAD v1.1 f1 score of 90.874% for the BERT-Large PyTorch model following [this instruction](https://github.com/mlcommons/inference/tree/master/language/bert#commands).
(+ upgrading 'transformers' to 4.0.0 to avoid the following error.
```
Traceback (most recent call last):
  File "run.py", line 120, in <module>
    main()
  File "run.py", line 101, in main
    lg.StartTestWithLogSettings(sut.sut, sut.qsl.qsl, settings, log_settings)
  File "/workspace/pytorch_SUT.py", line 70, in issue_queries
    start_scores = model_output.start_logits
AttributeError: 'tuple' object has no attribute 'start_logits'
Makefile:149: recipe for target 'run_pytorch_accuracy' failed
```
)

But, I am getting the F1 score of less than 90 only as follows.
```
$ make run_pytorch_accuracy
...
Reading examples...
Loading cached features from 'eval_features.pickle'...
Loading LoadGen logs...
Post-processing predictions...
Writing predictions to: build/result/predictions.json
Evaluating predictions...
{"exact_match": 82.26111636707664, "f1": 89.91125746620186}
```
I'd appreciate if you could let me know any clue or solution to the 90.874 F1 score.
I have no problem with the TensorFlow BERT-Large model.

--- a/language/bert/pytorch_SUT.py
+++ b/language/bert/pytorch_SUT.py
@@ -25,6 +25,7 @@ sys.path.insert(0, os.getcwd())
 import mlperf_loadgen as lg
 import numpy as np
 import torch
+import transformers
 from transformers import BertConfig, BertForQuestionAnswering
 from squad_QSL import get_squad_QSL
 
@@ -48,11 +49,13 @@ class BERT_PyTorch_SUT():
             vocab_size=config_json["vocab_size"])
 
         self.dev = torch.device("cuda:0") if torch.cuda.is_available() else torch.device("cpu")
+        self.version = transformers.__version__
 
         print("Loading PyTorch model...")
         self.model = BertForQuestionAnswering(config)
         self.model.to(self.dev)
-        self.model.load_state_dict(torch.load("build/data/bert_tf_v1_1_large_fp32_384_v2/model.pytorch"), strict=False)
+        self.model.eval()
+        self.model.load_state_dict(torch.load("build/data/bert_tf_v1_1_large_fp32_384_v2/model.pytorch"), strict=True)
 
         print("Constructing SUT...")
         self.sut = lg.ConstructSUT(self.issue_queries, self.flush_queries, self.process_latencies)
@@ -67,8 +70,11 @@ class BERT_PyTorch_SUT():
                 model_output = self.model.forward(input_ids=torch.LongTensor(eval_features.input_ids).unsqueeze(0).to(self.dev),
                     attention_mask=torch.LongTensor(eval_features.input_mask).unsqueeze(0).to(self.dev),
                     token_type_ids=torch.LongTensor(eval_features.segment_ids).unsqueeze(0).to(self.dev))
-                start_scores = model_output.start_logits
-                end_scores = model_output.end_logits
+                if self.version >= '4.0.0':
+                    start_scores = model_output.start_logits
+                    end_scores = model_output.end_logits
+                else:
+                    start_scores, end_scores = model_output
                 output = torch.stack([start_scores, end_scores], axis=-1).squeeze(0).cpu().numpy()
 
                 response_array = array.array("B", output.tobytes())

